k8s/service_controller/server/mem/in_mem_service_storage.go

```diff
@@ -79,8 +79,14 @@ func (s *InMemServiceStorage) AddBroker(broker *model.ServiceBroker, catalog *mo
 	return nil
 }
 
-func (s *InMemServiceStorage) DeleteBroker(string) error {
-	return fmt.Errorf("Not implemented yet")
+func (s *InMemServiceStorage) DeleteBroker(id string) error {
```

Comment:
Added a TODO to delete bindings. It's bit more complex and since nothing is persisted any ways, seems like not a big deal atm.